internal/monitor_conf.go

```diff
@@ -270,6 +270,13 @@ func GetMonitorProcessConfiguration(cluster *fdbtypes.FoundationDBCluster, proce
 		configuration.Arguments = append(configuration.Arguments, monitorapi.Argument{Value: fmt.Sprintf("--locality_data_hall=%s", cluster.Spec.DataHall)})
 	}
 
+	if cluster.UseDNSInClusterFile() {
+		configuration.Arguments = append(configuration.Arguments, monitorapi.Argument{ArgumentType: monitorapi.ConcatenateArgumentType, Values: []monitorapi.Argument{
+			{Value: "--locality_dns_name="},
```

Comment:
Is this an official locality that is required by FDB to make the DNS resolution work or is that only used by the operator?

Reply:
It is only used by the operator.

Reply:
I think we should document all "special" localities that the operator sets. That would be useful for others to e.g. query the status output.